internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
+	OpenAIToken string `env:"OPENAI_API_KEY,notEmpty"`
+}
+
+func LoadConfiguration() (Configuration, error) {
```

Comment:
Consider validating if the tokens are not empty after parsing the environment variables. The `env` package's validation may not cover all edge cases, and explicitly checking can help prevent issues in case the environment variables are set but empty.